app/bundles/DynamicContentBundle/Entity/DynamicContent.php

```diff
@@ -137,6 +149,17 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
 
         self::addTranslationMetadata($builder, self::class);
         self::addVariantMetadata($builder, self::class);
+        self::addFiltersMetadata($builder);
+
+        $builder->createField('isCampaignBased', 'boolean')
```

Comment:
Let's add an index on this (be sure to add it to the migration file as well)

Reply:
Done in latest commit

Reply:
Fixed